subprojects/tooling-api/src/main/java/org/gradle/tooling/TestLauncher.java

```diff
@@ -82,6 +82,8 @@
      */
     TestLauncher withJvmTestMethods(String testClass, Iterable<String> methods);
 
+    TestLauncher withDebugOptions(int port, boolean suspend);
```

Comment:
If we suspect additional arguments here, we might want a parameter object instead.

Reply:
For this story, I have a couple of open questions. I added my initial design and questions here: https://github.com/gradle/gradle/issues/9610